client/ayon_blender/api/plugin.py

```diff
@@ -202,37 +203,40 @@ def cache_instance_data(shared_data):
             cache = {}
             cache_legacy = {}
 
-            avalon_instances = bpy.data.collections.get(AVALON_INSTANCES)
-            avalon_instance_objs = (
-                avalon_instances.objects if avalon_instances else []
+            ayon_instances = bpy.data.collections.get(AYON_INSTANCES)
```

Comment:
Shouldn't we relink avalon instances into ayon instances and remove avalon collection afterwards?

Reply:
Yes, the idea should be that we "convert" the old data into the new data (early on) so that we can rely in most areas of the code on the new `AYON_INSTANCES` so we do not need to check for both everywhere.

Reply:
Is this part already implemented?

Reply:
Shouldn't `convert_avalon_instances` be called before `ayon_instances` are received?